packages/opentelemetry-plugin-grpc-js/src/client/utils.ts

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GrpcJsPlugin } from '../grpcJs';
+import type { GrpcClientFunc, SendUnaryDataCallback } from '../types';
+import {
+  SpanKind,
+  Span,
+  CanonicalCode,
+  Status,
+  propagation,
+} from '@opentelemetry/api';
+import {
+  GeneralAttribute,
+  RpcAttribute,
+} from '@opentelemetry/semantic-conventions';
+import type * as grpcJs from '@grpc/grpc-js';
+import {
+  grpcStatusCodeToSpanStatus,
+  findIndex,
+  grpcStatusCodeToCanonicalCode,
+  CALL_SPAN_ENDED,
+} from '../utils';
+
+export function getMethodsToWrap(
+  client: typeof grpcJs.Client,
+  methods: { [key: string]: { originalName?: string } }
+): string[] {
+  const methodsToWrap = [
+    ...Object.keys(methods),
+    ...(Object.keys(methods)
+      .map(methodName => methods[methodName].originalName)
+      .filter(
+        originalName =>
+          // eslint-disable-next-line no-prototype-builtins
```

Comment:
should we disable this rule completely ?

Reply:
Hm, do you mean for the entire project? I am not opposed... I think the only benefit it has for this project is to gently discourage using builtins unless necessary

Reply:
we are patching many things, so I guess this might be not needed rule at all for this project, this is just my thought :)